source/system-administrators/activities/encrypting-configuration-properties.rst

```diff
@@ -0,0 +1,52 @@
+:is-up-to-date: True
+
+.. index:: Encrypting Configuration Properties
+
+.. _encrypting-configuration-properties:
+
+===================================
+Encrypting Configuration Properties
+===================================
+
+It's recommended that configuration properties like access keys or passwords are encrypted since they contain sensible 
+data that shouldn't be publicly available to anyone but developers and administrators. In order to do that, follow the 
+next steps (you need a system administrator for the first step):
+
+.. WARNING :: Please do not use the same password and salt shown in the example. You should generate your own.
+
+#. Configure encryption by making defining the following environment variables before starting Crafter:
```

Comment:
Typo?  Change `Configure encryption by making defining the following...` to `Configure encryption by defining the following...`